frontend/src/common/enums/worker/workers.enum.ts

```diff
@@ -0,0 +1,8 @@
+enum Page {
```

Comment:
```suggestion
enum Pagination {
```
not sure about the name. since we will use it not only for worker, you need to more into other folder